glide-core/src/client/value_conversion.rs

```diff
@@ -853,6 +853,16 @@ pub(crate) fn expected_type_for_cmd(cmd: &Cmd) -> Option<ExpectedReturnType> {
             key_type: &None,
             value_type: &None,
         }),
+        b"XCLAIM" => {
+            if cmd.position(b"JUSTID").is_some() {
+                Some(ExpectedReturnType::ArrayOfStrings)
+            } else {
+                Some(ExpectedReturnType::Map {
+                    key_type: &Some(ExpectedReturnType::BulkString),
```

Comment:
Maybe
```suggestion
                    key_type: &Some(ExpectedReturnType::SimpleString),
```
It is entry ID right? In timestamp-like format.